### ISSUE TITLE
Issue of deeplink opening app on modal

### ISSUE THREAD

**Issue by ceroles** [MicrosoftDocs/msteams-docs#9678]
We are currently using deeplinking in our Teams custom app for users to be able to open directly to a specific tab/page.

We have noticed strange behavior recently wherein triggering a deeplink while the Teams app is open in the background with About modal, the app opens on the modal rather than main window.

Attached video for reference:
https://github.com/MicrosoftDocs/msteams-docs/assets/118153014/b8a195ec-77a8-4093-9bea-34cd3191f8b9

Replicated on iPad iOS version 16.6, Teams versions 5.17.1

**Comment by Meghana-MSFT** [comment]
@ceroles - Thank you for reporting this, we will check this. Meanwhile, could you please share the deeplink that you are using? Also is the deeplink navigating correctly when the Teams app is not in background?

**Comment by ceroles** [comment]
Yes, it is opening correctly when Teams app not in background. The "About" modal has to be open first on the Teams app. We are using the following deeplink format: `https://teams.microsoft.com/l/entity/<appId>/<entityId>?context={"subEntityId":"<subEntityId>"}`. Thank you.

**Comment by Meghana-MSFT** [comment]
Thank you for the additional details, we will check this and get back to you.

**Comment by ceroles** [comment]
Yes, we have replicated in the following so far:
iPad iOS version 16.6, Teams versions 5.17.1
iPhone SE iOS version 16.6, Teams version 5.18.1.
Thank you.

**Comment by Meghana-MSFT** [comment]
Thank you for your inputs, we have raised a bug for this issue. We will keep you posted on the updates. Thank you.
